.github/ISSUE_TEMPLATE/release.md

```diff
@@ -18,31 +18,29 @@ Please assign the release manager to the issue.
 
 This issue can be closed when we have completed the following steps (in order).
 Please ensure all artifacts (PRs, workflow runs, Tweets, etc) are linked from
-this issue for posterity. Refer to this [prior release issue][release-1.7] for
-examples of each step.
+this issue for posterity. Refer to this [prior release issue][release-1.11] for
+examples of each step, assuming release vX.Y.0 is being cut.
```

Comment:
We should also update the patch release issue template as part of this PR, so both are up to date :)

https://github.com/crossplane/crossplane/blob/master/.github/ISSUE_TEMPLATE/patch_release.md

Reply:
I gave it a shot, please have a look at it, I also updated/cleaned up the minor patch release doc once more.